src/sdk/pynni/nni/compression/torch/pruning/sensitivity_pruner.py

```diff
@@ -0,0 +1,368 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
if this is a new pruner, please update doc accordingly

Reply:
Updated~